env/staging/cwagent-fluentd-quickstart.yaml

```diff
@@ -70,7 +70,8 @@ data:
             "kubernetes":{
                 "cluster_name":"notification-canada-ca-staging-eks-cluster",
                 "metrics_collection_interval":60
-            }
+            },
+            "emf": { }
```

Comment:
I was expecting a format specification following the documentation. Is this just to start with for now?

Reply:
@jimleroyer it is indeed to just start. We haven't settled on a format yet. This will allow us to try different metrics and circle back once we know which key's we'll use.